assets/Script/technology/techGame1.js

```diff
@@ -0,0 +1,22 @@
+cc.Class({
+  extends: cc.Component,
+
+  properties: {
+    canvas: cc.Node
+  },
+
+  // LIFE-CYCLE CALLBACKS:
+  onLoad() {
+    this.canvas.opacity = 0;
+    this.canvas.color = new cc.Color(0, 0, 0);
+    this.canvas.runAction(
+      cc.fadeIn(2.0)
```

Comment:
Hey Jesse, maybe we could make the fade in time a global variable so that it's the same everywhere?